packages/esm-appointments-app/src/appointments/details/appointment-details.component.tsx

```diff
@@ -63,7 +63,7 @@ const AppointmentDetails: React.FC<AppointmentDetailsProps> = ({ appointment })
               <span className={styles.historyGridCount}>{appointmentsCount.cancelledAppointments}</span>
             </div>
             <div>
-              <p className={styles.historyGridLabel}>{t('upcomming', 'Upcoming')}</p>
+              <p className={styles.historyGridLabel}>{t('upcoming', 'Upcoming')}</p>
```

Comment:
Not sure where this came from - this cropped up as message changes when I tried to commit, I noticed this typo, so I fixed it (which enabled us to not lose the existing translations).